tests/components/create_comment/__snapshots__/create_comment.test.jsx.snap

```diff
@@ -389,9 +389,9 @@ exports[`components/CreateComment should match snapshot, empty comment 1`] = `
         </span>
       </div>
     </div>
-    <MsgTyping
+    <Connect(MsgTyping)
```

Comment:
? why do you need a connect here, you already connceted it in msg_typing/index.js

Reply:
sorry, didn't realize I was looking at a snapshot file. ignore the comment